README.md

```diff
@@ -14,13 +14,21 @@ Tasks sent from a synchronous context to an asynchronous context in Swift Concur
 
 ```swift
 @MainActor
-func test_mainActor_taskOrdering() async {
-    var counter = 0
+func testMainActorTaskOrdering() async {
+    actor Counter {
```

Comment:
We don't _need_ a `Counter` type in this example but the below examples do need it. For the sake of consistency with the below additions I made this example test a dash more complicated.